Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/Pipeline/Forward/ForwardPassSrg.azsli

```diff
@@ -14,11 +14,16 @@ ShaderResourceGroup PassSrg : SRG_PerPass
 {
     // [GFX TODO][ATOM-2012] adapt to multiple shadowmaps
     Texture2DArray<float> m_directionalLightShadowmap;
-    Texture2DArray<float> m_directionalLightExponentialShadowmap;
     Texture2DArray<float> m_projectedShadowmaps;
+#if !defined(FORCE_SMALL_SRGS) || ENABLE_ESM_SHADOW
```

Comment:
Why have **!defined(FORCE_SMALL_SRGS)** in this check? It seems to complicate this check unnecessarily. We can use FORCE_SMALL_SRGS to disable ENABLE_ESM_SHADOW. Same goes for all the checks below.